docker-compose.yml

```diff
@@ -1,105 +1,32 @@
-version: '3'
+#docker compose -f docker-compose-3-persona.yml up --build -d
```

Comment:
Sorry, is this perhaps a leftover ?

Would it look nicer for this line to be like the line below ?

```
docker compose up -d # OR docker compose down -v --remove-orphans && docker compose up -d
```